package.json

```diff
@@ -40,8 +40,8 @@
     "@radix-ui/react-toggle-group": "^1.1.0",
     "@radix-ui/react-tooltip": "^1.1.0",
     "@radix-ui/react-visually-hidden": "^1.1.0",
-    "@renegade-fi/internal-sdk": "0.0.0-canary-20240829175257",
-    "@renegade-fi/react": "0.4.12",
+    "@renegade-fi/internal-sdk": "0.0.15",
+    "@renegade-fi/react": "0.0.0-canary-20250104172719",
```

Comment:
this is a canary release branched off of `0.4.13` which exposes the custom `useQuery` util